gemini/gim.py

```diff
@@ -302,25 +303,40 @@ def report_candidates(self):
                     yield item
 
     def run(self):
+        test = []
+        res = []
         has_gts = False
         from .gemini_bcolz import gt_cols_types
         for i, s in enumerate(self.report_candidates()):
             if i == 0:
                 has_gts = [x[0] for x in gt_cols_types if x[0] in s] or False
-                print("\t".join(s.keys()))
+                if self.args.format is 'default': print("\t".join(s.keys()))
```

Comment:
don't use `is` for string equality. use `==`. It will probably always work due to interning, but makes me nervous.